src/Client.php

```diff
@@ -36,12 +36,16 @@ public function getDaily(\DateTime $start, \DateTime $end, array $stations, arra
         return (new ResponseParser())->parse($this->httpClient->sendRequest($request)->getBody()->getContents());
     }
 
-    public function getHourly(\DateTime $start, \DateTime $end, array $stations, array $vars) : array
+    public function getHourly(\DateTime $start, \DateTime $end, $timeframe_start, $timeframe_end, array $stations, array $vars) : array
```

Comment:
Could you change the new variables to $camelCase?

Reply:
Check!